sdk/keyvault/azure-keyvault-administration/CHANGELOG.md

```diff
@@ -16,9 +17,14 @@
 - The `KeyVaultSetting` model's `type` parameter and attribute have been renamed to `setting_type`
 - The `SettingType` enum has been renamed to `KeyVaultSettingType`
 
-### Bugs Fixed
-
 ### Other Changes
+- (From 4.3.0b1) Python 3.6 is no longer supported. Please use Python version 3.7 or later.
+- Key Vault API version `7.4` is now the default
+- (From 4.3.0b1) Updated minimum `azure-core` version to 1.24.0
+- (From 4.3.0b1) Dropped `msrest` requirement
+- (From 4.3.0b1) Dropped `six` requirement
+- (From 4.3.0b1) Added requirement for `isodate>=0.6.1` (`isodate` was required by `msrest`)
+- (From 4.3.0b1) Added requirement for `typing-extensions>=4.0.1`
```

Comment:
Nit: I'd group all the "(From 4.3.0b1)"s together. Easy to miss line 22 in the sea of those changes otherwise.